jni/src/faiss_index_service.cpp

```diff
@@ -329,16 +329,14 @@ void ByteIndexService::insertToIndex(
 
     // Add vectors in batches by casting int8 vectors into float with a batch size of 1000
     int batchSize = 1000;
-    std::vector <float> inputFloatVectors(batchSize * dim);
-    std::vector <int64_t> floatVectorsIds(batchSize);
+    float inputFloatVectors[batchSize * dim];
```

Comment:
Try using malloc here to create the array.

Reply:
malloc is also working and I have verified with a benchmarking test. But, going with std::vector changes as either of them works without resize. Thanks!